frontend/src/bundles/landing/components/feedbacks-part/styles.module.scss

```diff
@@ -1,77 +1,88 @@
-.container {
+.body {
     display: flex;
     flex-direction: column;
-    justify-content: space-around;
+    justify-content: center;
     align-items: center;
+    row-gap: 20px;
     width: 100%;
-    height: 620px;
-    padding: var(--p-5) 0;
     background-color: var(--color-white-100);
 }
 
+.container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    max-width: var(--container);
+    padding: 90px var(--p-3);
+}
+
 .carouselContainer {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: repeat(3, minmax(288px, 360px));
+    gap: var(--m-5);
+    padding-bottom: 60px;
+}
+
+.bulletClass {
+    display: inline-block;
+    width: 8px;
+    height: 8px;
+    background-color: var(--color-gray-100);
+    border-radius: 50%;
+}
+
+.bulletActive {
+    background-color: var(--color-turquoise-500);
+}
+
+.horizontalClass {
     display: flex;
-    flex-direction: row;
-    column-gap: 50px;
+    justify-content: center;
+    gap: 10px;
+}
+
+.carouselContainer > div {
+    width: 100%;
 }
 
 .title {
+    margin-top: 0;
     color: var(--color-navy-900);
     font-size: var(--font-size-level-1-heading);
     line-height: var(--line-height-level-1-heading);
 }
 
-@media (max-width: 768px) {
+@media (max-width: 1200px) {
```

Comment:
Replace with variable

Reply:
Resolved in [d153e37](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/d153e379cd5c556a068d7900f44e914546b8ee72)